src/components/TextArea/TextArea.css

```diff
@@ -0,0 +1,7 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/base-text-field.css";
```

Comment:
we're essentially still duplicating the BaseTextField CSS file by importing its contents into both TextArea and TextField. if we're taking this approach instead of composition it is probably better for TextArea.tsx and TextField.tsx to import 'base-text-field.css' so webpack can include it once.